apps/frontend/src/pages/register.tsx

```diff
@@ -166,6 +169,22 @@ const Register: React.FC = () => {
   };
 
   const handleRegisterWithPassword = async (password: string) => {
+
+    // Check password strength
+    const passwordCheck = zxcvbn(password);
+    if (passwordCheck && passwordCheck.score < 3) {
```

Comment:
why 3? seems arbitrary, can you add a comment and link to some docs explaining this?